src/dbnode/storage/database_test.go

```diff
@@ -21,13 +21,16 @@
 package storage
 
 import (
+	stdlib "context"
 	"errors"
 	"fmt"
 	"sort"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/m3db/m3/src/dbnode/tracepoint"
```

Comment:
import order. interestingly, did metalint not pick this up?

Reply:
weird - ill look into it